Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+pipeline {
+    agent none
+    environment {
+        RUST_VERSION='1.50.0'
+        RUST_NIGHTLY='nightly-2021-02-21'
+    }
+    options{
+        lock('singleton-build') 
+    }
+    stages {
+        stage('Prepare and Build with extra features') {
+            agent {
+                docker {
+                    label 'benchmark'
+                    image 'rustlang/rust:nightly'
+                    args '-u root:root'
+                }
+            }
+            steps {
+                sh 'lscpu'
+                sh 'free -h'
+                sh '''
+                  apt-get update && \
+                  apt-get -y install apt-utils cmake pkg-config libssl-dev git clang libclang-dev && \
+                  rustup uninstall nightly && \
+                  rustup install $RUST_VERSION && \
+                  rustup install $RUST_NIGHTLY && \
+                  rustup default $RUST_VERSION && \
+                  rustup target add --toolchain $RUST_NIGHTLY wasm32-unknown-unknown && \
+                  rustup target add --toolchain $RUST_VERSION x86_64-unknown-linux-musl && \
+                  mv /usr/local/rustup/toolchains/nightly* /usr/local/rustup/toolchains/nightly-x86_64-unknown-linux-gnu
+                '''
+                sh 'cargo build --release --features runtime-benchmarks'
+		sh 'cargo test -p crml-staking --features runtime-benchmarks'
```

Comment:
whitespace
```suggestion
                sh 'cargo build --release --features runtime-benchmarks'
                sh 'cargo test -p crml-staking --features runtime-benchmarks'
```

Reply:
I will remove those spaces in a new commit.